meta/runtime.yml

```diff
@@ -63,3 +65,12 @@ plugin_routing:
           deprecation:
             removal_version: 3.0.0
             warning_text: Use infoblox.bloxone.dns_auth_zone_info instead.
+
+        b1_dns_forward_zone:
```

Comment:
Please remove this, since these modules don't exist. 

Also, above says `b1_dns_auth_zone_gather`, but I only see b1_dns_zone_gather in master. Maybe that should be corrected to show it has been replaced with the two new modules.

Reply:
done

Reply:
Changelog here should also be fixed, but can be done in a different PR as well. - https://github.com/infobloxopen/bloxone-ansible/blob/v2/changelogs/fragments/35-auth-zone.yml#L3

Reply:
done